### ISSUE TITLE
[5.0.0-beta.1] Is bootstrap css auto-imported? Can I use boostrap without a CDN?

### ISSUE THREAD

**Issue by RudolfVonKrugstein** [bestguy/sveltestrap#222]
I am trying out the `5.0.0-beta.1` branch.

I installed `sveltestrap@5.0.0-beta.1`. Than, without adding the boostrap css import, I tried out the dev server and noticed:

`boostrap.min.css` is already loaded from `https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css`.

In the documentation, it says, that I have to import the boostrap css. Is that still true? Because I did not do that.

Also: Can I prevent the loading of `boostrap.min.css` via CDN? For data privacy reasons I would prefer to serve it from my own server.

Thanks! And Thanks for the great project.

**Comment by RudolfVonKrugstein** [comment]
I do not know what I did wrong, but my first problem vanished. I re-created the project and bootstrap.min.css does not get loaded automatically anymore.

For importing bootstrap without a CDN, I did the following:

I installed bootstrap via npm:

```
npm install bootstrap@next
```

And than I followed this stackoverflow post: https://stackoverflow.com/questions/63299785/how-to-setup-global-bootstrap-via-scss-in-svelte to use the scss version of bootstrap.

Sorry for the noise.
